pkg/config/linters_settings.go

```diff
@@ -376,6 +376,7 @@ type ReviveSettings struct {
 	IgnoreGeneratedHeader bool `mapstructure:"ignore-generated-header"`
 	Confidence            float64
 	Severity              string
+	EnableAllRules        bool `mapstructure:"enable-all-rules"`
```

Comment:
Should that be documented in https://github.com/golangci/golangci-lint/blob/master/.golangci.example.yml?

Reply:
yes I am working to update the revive section in the example file.

Reply:
It is still not documented

Reply:
https://github.com/golangci/golangci-lint/pull/2458